training-modules.Rproj

```diff
@@ -0,0 +1,13 @@
+Version: 1.0
```

Comment:
The short answer is I don't know and [this](https://community.rstudio.com/t/should-nested-projects-be-avoided-and-if-so-why/58686) is what I've been able to find about nested projects in general. If I recall correctly, the reason we have a pathway analysis project is because we like to use them for the teaching of training itself. In my mind, the `training-modules.Rproj` would be for development purposes only really and we could 1. get rid of (and ignore!) all the other subprojects 2. if we really need them _during training_ can we script project creation (don't know if this is feasible) as part of getting better at training data management, etc. - thoughts?